docs/croissant_rai.ttl

```diff
@@ -0,0 +1,127 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix croissant: <http://mlcommons.org/croissant/> .
+@prefix rai: <http://mlcommons.org/croissant/RAI/> .
+
+# Dataset level properties
+
+rai:dataCollection a rdf:Property ;
+rdfs:label "dataCollection" ;
+rdfs:comment "Description of the data collection process." ;
+schema:domainIncludes schema:Dataset ;
+schema:rangeIncludes schema:Text .
+
+rai:dataCollectionType a rdf:Property ;
+rdfs:label "dataCollectionType" ;
+rdfs:comment "Define the data collection type. Recommended values: Surveys, Secondary Data analysis, Physical data collection, Direct measurement, Document analysis, Manual Human Curator, Software Collection, Experiments, Web Scraping, Web API, Focus groups, Self-reporting, Customer feedback data, User-generated content data, Passive Data Collection, Others." ;
+schema:domainIncludes schema:Dataset ;
+schema:rangeIncludes schema:Text, schema:ItemList .
```

Comment:
How should ItemList be used for values of this property? Alternatively, values can just be a list of strings, e.g.:

```
"dataCollectionType": ["Surveys", "Secondary Data analysis"]
```

Following schema.org, this is allowed if you just specify the range to be Text.

This comment applies to all the ItemList instances in this turtle file.

Reply:
We do not have a complete taxonomy of collection types. We discussed giving some recommended options but allowing users freedom (also following the schema common criteria). This freedom will let us look at users' behavior in the future and maybe recommend a more closed item list.  

So, range could be schema:Text

Reply:
I copied the `range` from `ExpectedType column` of the table "RAI property information" present in the RAI spec.

Reply:
I added suggestions to update the spec.